h2o-algos/src/main/java/hex/tree/xgboost/XGBoostGPULock.java

```diff
@@ -0,0 +1,21 @@
+package hex.tree.xgboost;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class XGBoostGPULock {
```

Comment:
Do you need to publish this one outside of xgboost package, if i understand well this is just technical aspect of implementation. Can be package private.